pkg/fanal/analyzer/secret/secret_test.go

```diff
@@ -14,6 +15,8 @@ import (
 )
 
 func TestSecretAnalyzer(t *testing.T) {
+	t.Skip("skipping test in short mode.")
```

Comment:
Don't we need `testing.Short()`?

```suggestion
	if testing.Short() {
		t.Skip("skipping test in short mode.")
	}
```